plugins/sip/lib/model/providers/pexip/kPexipUtils.php

```diff
@@ -423,6 +423,22 @@ public static function extractIdFromCreatedResult($result,$url ,$headerSize)
 				return $virtualRoomId;
 			}
 		}
+
```

Comment:
To avoid code duplication, create protected function getVirtualRoomId($url);
if it does not find return null.
And then call it once with the full URL if not fallback to the path only

Reply:
@MosheMaorKaltura @gotlieb , Changes made, please review.